src/VirtoCommerce.SimpleExportImportModule.Data/Validation/ImportProductPricesExistanceValidator.cs

```diff
@@ -0,0 +1,41 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentValidation;
+using FluentValidation.Validators;
+using VirtoCommerce.PricingModule.Core.Model.Search;
+using VirtoCommerce.PricingModule.Core.Services;
+using VirtoCommerce.SimpleExportImportModule.Core.Models;
+
+namespace VirtoCommerce.SimpleExportImportModule.Data.Validation
+{
+    public class ImportProductPricesExistenceValidator: AbstractValidator<ImportProductPrice[]>
```

Comment:
Add space before the colon.
P.S. And please enable autoformatter ;)

Reply:
It's enabled :) For some reason ReSharoper doesn't apply these rules to newly generated class

Reply:
Fixed